apps/spruce/src/analytics/version/useVersionAnalytics.ts

```diff
@@ -62,10 +62,12 @@ export const useVersionAnalytics = (id: string) => {
       fetchPolicy: "cache-first",
     },
   );
-  const { status } = eventData?.version || {};
+  const { isPatch, requester, status } = eventData?.version || {};
 
   return useAnalyticsRoot<Action, AnalyticsIdentifier>("Version", {
     "version.status": status || "",
     "version.id": id,
+    "version.is_patch": isPatch || false,
+    "version.requester": requester || "",
```

Comment:
should we track `requester` for `usePatchAnalytics` or does it not matter for them?

Reply:
I don't think its as valuable as it is for version interactions. We also don't currently return requester for patches in our API so we could add it down the line but outside of the scope of this ticket.